src/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -269,19 +269,136 @@ public async Task it_returns_diagnostics(Language language)
 
             await SubmitCode(kernel, source);
 
-            var error = language switch
-            {
-                Language.FSharp => "input.fsx (1,1)-(1,7) typecheck error The value or constructor 'aaaadd' is not defined.",
-                Language.CSharp => "(1,1): error CS0103: The name 'aaaadd' does not exist in the current context"
-            };
+            var diagnosticRange = new LinePositionSpan(
+                new LinePosition(0, 0),
+                new LinePosition(0, 6));
 
             KernelEvents
                 .Should()
                 .ContainSingle<CommandFailed>()
                 .Which
                 .Message
                 .Should()
-                .Be(error);
+                .Be(errorMessage);
```

Comment:
don't think we need to assert on command filed for compile error, but if you do multiple assertions use the assertion scope